deploy/helm/sumologic/requirements.yaml

```diff
@@ -8,7 +8,7 @@ dependencies:
     repository: https://kubernetes-charts.storage.googleapis.com/
     condition: prometheus-operator.enabled
   - name: falco
-    version: 1.1.1
+    version: 1.1.0
```

Comment:
should also update the dependency table in master branch so it gets included when we cut the next release?